src/plugins/data/public/antlr/shared/constants.ts

```diff
@@ -167,26 +167,67 @@ export const testingIndex = ({
 } as unknown) as IndexPattern;
 
 export const booleanOperatorSuggestions = [
-  { text: 'or', type: 17 },
-  { text: 'and', type: 17 },
+  { end: -1, start: -1, text: 'or', type: 17 },
+  { end: -1, start: -1, text: 'and', type: 17 },
 ];
 
-export const notOperatorSuggestion = { text: 'not', type: 17 };
+export const notOperatorSuggestion = { text: 'not', type: 17, end: -1, start: -1 };
 
-export const fieldNameSuggestions: Array<{ text: string; type: number; insertText?: string }> = [
-  { text: 'Carrier', type: 3, insertText: 'Carrier: ' },
-  { text: 'DestCityName', type: 3, insertText: 'DestCityName: ' },
-  { text: 'DestCountry', type: 3, insertText: 'DestCountry: ' },
-  { text: 'DestWeather', type: 3, insertText: 'DestWeather: ' },
-  { text: 'DistanceMiles', type: 3, insertText: 'DistanceMiles: ' },
-  { text: 'FlightDelay', type: 3, insertText: 'FlightDelay: ' },
-  { text: 'FlightNum', type: 3, insertText: 'FlightNum: ' },
-  { text: 'OriginWeather', type: 3, insertText: 'OriginWeather: ' },
-  { text: '_id', type: 3, insertText: '_id: ' },
-  { text: '_index', type: 3, insertText: '_index: ' },
-  { text: '_score', type: 3, insertText: '_score: ' },
-  { text: '_source', type: 3, insertText: '_source: ' },
-  { text: '_type', type: 3, insertText: '_type: ' },
+export const fieldNameSuggestions: Array<{
```

Comment:
we shouldn't store test values in constants file. constants will get bundled up and will increase the release distribution size. test files do not get compiled with the distribution.

Reply:
out of curiosity, could you point me to where its specified which files don't get compiled in the distribution? would it just be .test.ts files, or are there other files that wouldn't be included?

Reply:
there are some hard coded ignores in the repo, like https://github.com/opensearch-project/OpenSearch-Dashboards/blob/25c23db905e1b0f8a83229def41151a6e275f097/packages/osd-pm/src/utils/targeted_build.ts#L58-L59